website/examples/README.md

```diff
@@ -2,6 +2,8 @@
 <img src="../../media/examples.png" width="400">
 </p>
 
+# Examples
```

Comment:
Is it possible to include all available notebooks into this tab (and not only `CCSD` and `DOCI`) and maybe move it to a separate folder?